jobs/kola-aws.Jenkinsfile

```diff
@@ -74,27 +74,42 @@ try { timeout(time: 90, unit: 'MINUTES') {
 
         withCredentials([file(variable: 'AWS_CONFIG_FILE',
                               credentialsId: 'aws-kola-tests-config')]) {
-            fcosKola(cosaDir: env.WORKSPACE, parallel: 5,
-                     build: params.VERSION, arch: params.ARCH,
-                     extraArgs: params.KOLA_TESTS,
-                     skipBasicScenarios: true,
-                     platformArgs: '-p=aws --aws-region=us-east-1')
+            // A few independent tasks that can be run in parallel
+            def parallelruns = [:]
+
+            parallelruns['Kola:Full'] = {
+                fcosKola(cosaDir: env.WORKSPACE, parallel: 5,
+                         build: params.VERSION, arch: params.ARCH,
+                         extraArgs: params.KOLA_TESTS,
+                         skipBasicScenarios: true,
+                         platformArgs: '-p=aws --aws-region=us-east-1')
+            }
 
             if (params.ARCH == "x86_64") {
                 def tests = params.KOLA_TESTS
                 if (tests == "") {
                     tests = "basic"
                 }
-                parallel Xen: {
+                parallelruns['Kola:Xen'] = {
                     // https://github.com/coreos/fedora-coreos-tracker/issues/997
+                    // Run this test on i3.large so we can also run ext.config.platforms.aws.nvme
+                    // to verify access to instance storage nvme disks works
+                    // https://github.com/coreos/fedora-coreos-tracker/issues/1306
+                    // Also add in the ext.config.platforms.aws.assert-xen test just
+                    // to sanity check we are on a Xen instance.
+                    def xen_tests = tests
+                    if (xen_tests == "basic") {
+                        xen_tests = "basic ext.config.platforms.aws.nvme ext.config.platforms.aws.assert-xen"
+                    }
                     fcosKola(cosaDir: env.WORKSPACE,
                              build: params.VERSION, arch: params.ARCH,
-                             extraArgs: tests,
+                             extraArgs: xen_tests,
                              skipUpgrade: true,
                              skipBasicScenarios: true,
-                             marker: "kola-m4",
-                             platformArgs: '-p=aws --aws-region=us-east-1 --aws-type=m4.large')
-                }, m6i: {
+                             marker: "kola-i3",
```

Comment:
Can we have these makers match the stage name? That way it's easier for someone less familiar with AWS to download the right artifact after seeing the failing stage from the pipeline artifact.

E.g. `kola-xen`, `kola-intel-ice-lake`, and `kola-graviton`?